apps/full-site-editing/package-lock.json

```diff
@@ -0,0 +1,2572 @@
+{
```

Comment:
I don't think this file is needed

Reply:
It's automatically generated when installing dependencies via CLI.
Even though it messes with the LOC diff, it's useful in case of package versions conflicts and whatnot.

Reply:
You can run `npm i --no-package-lock` to not generate lockfiles. https://docs.npmjs.com/cli/install

If it's useful, you should commit it to the repo and use `npm ci`. :-)

Reply:
> It's automatically generated when installing dependencies via CLI.

Did this happen when installing dependencies from the Calypso root or within the package folder? If it's the later, I was thinking that it is not necessary to install them because the package is actually not used in Calypso and we're not going to use those dependencies anywhere. 

But giving a second thought, maybe the build generation will fail if it cannot find the dependencies that are not externalized. So maybe we really need to install the dependencies?